src/app/public/modules/tabs/tab-dropdown.component.scss

```diff
@@ -2,48 +2,28 @@
 @import "~@skyux/theme/scss/_compat/mixins";
 
 .sky-tab-dropdown {
-  ::ng-deep .sky-dropdown-item {
-    margin: 0;
+  ::ng-deep {
+    .sky-dropdown-item {
+      margin: 0;
+    }
   }
 }
 
-.sky-tab-dropdown-button {
-  max-width: 300px;
-}
-
 .sky-tab-dropdown-item {
-  display: flex;
-  padding: 0;
-  transition: background-color $sky-transition-time-short;
-  min-width: 300px;
-
-  &:hover:not(.sky-tab-dropdown-item-selected) {
-    background-color: $sky-background-color-neutral-light;
+  sky-tab-button {
+    display: flex;
+    min-width: 300px;
   }
-}
-
-.sky-tab-dropdown-item-btn {
-  border: none;
-  background-color: transparent;
-  cursor: pointer;
-  flex-grow: 1;
-  text-align: left;
-}
-
-.sky-tab-dropdown-item-close {
-  background-color: transparent;
-  border: none;
-}
 
-.sky-tab-dropdown-item-selected {
-  background-color: $sky-background-color-primary-dark;
-
-  &:hover {
-    background-color: $sky-background-color-primary-dark;
-  }
+  ::ng-deep {
+    .sky-btn-tab {
+      margin: 0;
+      border-radius: 0;
+      width: 100%;
 
-  .sky-tab-dropdown-item-btn,
-  .sky-tab-dropdown-item-close {
-    color: #fff;
+      > .sky-btn-tab-close {
+        float: right;
```

Comment:
Floating can be really problematic - is there any way we could achieve this with flex instead?